docs/settings.rst

```diff
@@ -51,16 +51,30 @@ These settings can be used to change who can access the helpdesk.
 
   **Default:** ``HELPDESK_REDIRECT_TO_LOGIN_BY_DEFAULT = False``
 
-- **HELPDESK_VALID_EXTENSIONS** Valid extensions for file types that can be attached to tickets. Note: This used to be calle **VALID_EXTENSIONS** which is now deprecated.
+- **HELPDESK_ANON_ACCESS_RAISES_404** If True, redirects user to a 404 page when attempting to reach ticket pages while not logged in, rather than redirecting to a login screen.
 
-  **Default:** ``HELPDESK_VALID_EXTENSIONS = ['.txt', '.asc', '.htm', '.html', '.pdf', '.doc', '.docx', '.odt', '.jpg', '.png', '.eml']
+  **Default:** ``HELPDESK_ANON_ACCESS_RAISES_404 = False``
 
-- **HELPDESK_VALIDATE_ATTACHMENT_TYPES** If you'd like to turn of filtering of helpdesk extension types you can set this to False.
+Settings related to attachments:
 
-- **HELPDESK_ANON_ACCESS_RAISES_404** If True, redirects user to a 404 page when attempting to reach ticket pages while not logged in, rather than redirecting to a login screen.
+- **HELPDESK_ENABLE_ATTACHMENTS** If set to ``True``, files can be
+  attached to tickets and followups, and emails are searched for
+  attachments which are then attached to the ticket.  Also enables the
+  ``HELPDESK_ALWAYS_SAVE_INCOMING_EMAIL_MESSAGE`` setting.
 
-  **Default:** ``HELPDESK_ANON_ACCESS_RAISES_404 = False``
+  **Caution**: Set this to False, unless you have secured access to
```

Comment:
This is false FUD. Unless you have a non-standard Media setup that allows file listing, attachment names are protected by the ticket's secret key and cannot be guessed:

https://github.com/django-helpdesk/django-helpdesk/blob/c77bb30035bfc32d00dc6aab635b38f18e6538c4/helpdesk/models.py#L1218